modules/ROOT/pages/fd-publish-app-as-template.adoc

```diff
@@ -0,0 +1,43 @@
+= Share a Mule Application as a Template on Exchange
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can publish a Mule application as a template on Exchange, which makes it easy for you and other people to create similar Mule applications in the future.
+
+== Benefits of sharing a Mule application as a template
```

Comment:
Benefits of sharing a Mule Application as a Template

Reply:
H2 headings use sentence-style capitalization.